roles/sap_ha_install_hana_hsr/defaults/main.yml

```diff
@@ -1,19 +1,24 @@
 ---
-
-
 sap_ha_install_hana_hsr_sid: "{{ sap_hana_sid }}"
 sap_ha_install_hana_hsr_instance_number: "{{ sap_hana_instance_number }}"
-sap_ha_install_hana_hsr_db_system_password: ""
-sap_ha_install_hana_hsr_alias: ""
-sap_ha_install_hana_hsr_primary_ip: ""
-sap_ha_install_hana_hsr_primary_hostname: ""
-sap_ha_install_hana_hsr_secondary_ip: ""
-sap_ha_install_hana_hsr_secondary_hostname: ""
-sap_ha_install_hana_hsr_fqdn: ""
+sap_ha_install_hana_hsr_cluster_nodes: "{{ sap_hana_cluster_nodes }}"
 
-sap_ha_install_hana_hsr_hdbuserstore_system_backup_user: "HDB_USER"
+sap_ha_install_hana_hsr_hdbuserstore_system_backup_user: HDB_SYSTEMDB
+sap_ha_install_hana_hsr_db_system_password: "{{ sap_hana_install_master_password }}"
+sap_ha_install_hana_hsr_fqdn: "{{ sap_domain }}"
 sap_ha_install_hana_hsr_rep_mode: sync
 sap_ha_install_hana_hsr_oper_mode: logreplay
 
-# Specific variables per node
-sap_ha_install_hana_hsr_role: ""
+sap_ha_install_hana_hsr_update_etchosts: yes
+
+sap_ha_install_hana_hsr_configure_firewall: no
+sap_ha_install_hana_hsr_fw_ports:
```

Comment:
1. HANA ports can be configured differently on installed SAP HANA system. For SAP HANA there is no rule that pors are 3XX13 or 3XX15, where XX is instance number. Port configuration can be changed in SAP HANA configuration.
2. I think we should not include any firewall configuration in this role. One role - one function.

Reply:
At the moment the firewall configuration does not fully work as expected during our tests, hence default is off. We will work on improvements to provide this security feature in a reliable way.

Reply:
And because you are right, I have removed the defaults and the task + file.